.github/dependabot.yml

```
@@ -19,3 +19,11 @@ updates:
   - dependency-name: lxml
     versions:
     - 4.6.2
+- package-ecosystem: github-actions
+  directory: /
+  groups:
+    github-actions:
+      patterns:
+        - "*"  # Group all Actions updates into a single larger pull request
+  schedule:
+    interval: weekly
```

Monthly is less noisy:
```suggestion
    interval: monthly
```

---

It has been three weeks...

GitHub Actions have very infrequent major version changes.  For instance, `setup-python` has only had four major upgrades in its lifetime.  Also, when they are upgraded, it often happens in batches.  The `pattern: *` will consolidate all GHA updates into a single pull request to further reduce chattiness.  See: rapidfuzz/RapidFuzz#362

There is a tradeoff between supply chain security and chattiness.  Given that we have a few GHAs that are updated rarely and usually in batches and we are using `pattern: *` to ensure that there will only ever be a single GHA upgrade PR at a time, my recommendation would have been to kick off this process in weekly mode.  It is trivial to switch to monthly mode if we sense it becomes chatty.  I doubt that it will.

---

Perhaps the `pattern: *` approach should be used on the `pip` dependencies which are far more chatty than the GHA updates could ever be.  https://github.com/python/pythondotorg/pulls/app%2Fdependabot